test/integration/improve_this_page_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class ImproveThisPageTest < ActionDispatch::IntegrationTest
```

Comment:
I definitely wrote that test once.. I've lost it somehow. I agree it's needed. Ta.